docs/prow/prow-installation-on-forks.md

```diff
@@ -0,0 +1,191 @@
+
+3. Run the [`provision-cluster.sh`](../../development/provision-cluster.sh) script or follow [this](https://github.com/kubernetes/test-infra/blob/master/prow/getting_started_deploy.md#create-the-cluster) instruction to provision a new cluster on GKE. Ensure that kubectl points to the correct cluster. For GKE, execute the following command:
+
+```
+gcloud container clusters get-credentials $CLUSTER_NAME --zone=$ZONE --project=$PROJECT
+```
+
+## Create a bot account
+
+Create a separate GitHub account which serves as a bot account that triggers the Prow comments that you enter in the pull request. If the Prow bot account is the same as the account that creates a job-triggering comment, the job is not triggered.
+
+Add the bot account to the [collaborators](https://help.github.com/articles/adding-outside-collaborators-to-repositories-in-your-organization/) on your personal GitHub account.
+
+## Set an access token
+
+Set an OAuth2 token that has the read and write access to the bot account.
+
+To generate a new token, go to the **Settings** tab of a given GitHub account and click **Developer Settings**. Choose **Personal Access Token** and **Generate New Token**.
+In the new window, select all scopes and click **Generate token**.
+
+You can set the token either as an environment variable named `OAUTH` or provide it during the installation process when prompted.
+
+## Create Secrets
+
+For the purpose of the installation, you require to have a set of service accounts and secret files created on Google Cloud Storage (GCS).
+
+> **NOTE:** For details, see the [prow-secrets-management.md](./prow-secrets-management.md) document that explains step by step how to create all required GSC resources.
+
+1. Create two buckets on GCS, one for storing secrets and the second for storing logs.
+
+>**NOTE:** The bucket for storing logs is used in Prow by the Plank component. This reference is defined in the `config.yaml` file.
+
+2. Create the following service accounts, role bindings, and private keys, encrypt them using Key Management Service (KMS), and upload them to your Secret storage bucket:
+
+- **sa-gke-kyma-integration** with roles that allow the account to create Kubernetes clusters:
+    - Kubernetes Engine Cluster Admin (`roles/container.clusterAdmin`)
+    - Service Account User (`roles/iam.serviceAccountUser`)
+- **sa-vm-kyma-integration** with roles that allow the account to provision virtual machines:
+    - Compute Instance Admin (beta) (`roles/compute.instanceAdmin`)
+    - Compute OS Admin Login (`roles/compute.osAdminLogin`)
+    - Service Account User (`roles/iam.serviceAccountUser`)
+- **sa-gcs-plank** with the role that allows the account to store objects in a Bucket:
+    - Storage Object Admin (`roles/storage.objectAdmin`)
+- **sa-gcr-push** with the role that allows the account to push images to Google Container Repository:
+    - Storage Admin `roles/storage.admin`
+
+See an example of variables you need to export for each account:
```

Comment:
I think that example should be moved to `prow-secrets-management.md` instruction. These variables are not used in scripts used in this instruction.